system_linux_pc.cfg

```diff
@@ -22,3 +22,4 @@ r_ShadersAllowCompilation = 1
 
 -- Display FPS
 r_displayInfo = 3
+r_ShaderCompilerServer=127.0.0.1
```

Comment:
Is this change part of this PR?

Reply:
`git commit -a` strikes again. Will remove.